hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/ClassPathFormattingTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.util.phonehome;
+
+import org.junit.Test;
+
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static java.lang.String.join;
+import static org.junit.Assert.assertEquals;
+
+public class ClassPathFormattingTest {
+
+    public static final String PATH_SEPARATOR = System.getProperty("path.separator");
```

Comment:
There are convenient static vars for that purpose: `File.pathSeparator` and `File.separator`